src/theme/foundations/colors.ts

```diff
@@ -56,8 +59,9 @@ const colors: Colors = {
     },
     focus: "#4181F1",
     link: {
-      primary: "#0576D3",
-      secondary: "#004B98",
+      primary: linkPrimary,
+      secondary: linkSecondary,
+      wash: hexToRGB(linkPrimary, 0.05),
```

Comment:
Why is this called "wash"? Just curious.

Reply:
This color value is a transparency and I was trying to be semantic, but this could be better.  I will establish a more logical pattern that we can use for future colors with various transparency values.